sugarcrm/customer.py

```diff
@@ -0,0 +1,36 @@
+import sugarcrm
+from sugarcrm.client import APIClient
+
+
+class Customer:
+    def __init__(self, logger=sugarcrm.logger):
+        self.logger = logger
+        self.roles = set()
+        self.api_client = APIClient()
+
+    def fetch(self, email):
+        # In SugarCRM personal individuals are in Accounts module and organizations in Contacts
+        self.api_client.login()
```

Comment:
IMO this specific detail of the API fits better into `APIClient`. It knows already about the low-level details of the communication with SugarCRM. It could handle logging in if not done yet so consumers don't need to worry about it and duplicate this know-how.